### ISSUE TITLE
:wheelchair: a11y(switch radio buttons): focus lost after selection

### ISSUE THREAD

**Issue by carolabes** [baloise/design-system#1422]
### Description of this issue

* After selection, focus is lost.

### How to fix it

* Focus should remain, even after the selection.

### Additional Information

Level A
https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by GinaBiondo** [comment]
Apparently the issue is still present for keyboard navigation. When using the arrow keys to navigate between radio buttons the selection is made. Don't think this is correct. To select a radio button the space bar should be used if I am not mistaken. Arrow keys should be used to navigate between radio buttons and move the focus with the, Please check back with @carolabes on further details. THX!

**Comment by mmarinkov** [comment]
Hi @GinaBiondo-aperto 
Hi @carolabes 

I have some questions regarding the last comment. 

Initially we thought that #1422 was raised for the switch checkbox: 
- https://design.baloise.dev/?path=/story/components-form-checkbox--switch&globals=framework:angular. 
There we noticed that we are loosing focus and provided a fix for it. 

The last comment sounded to me as more radio button related. The issue with the radio buttons was also handled but in this task:  [#947](https://github.com/baloise/design-system/issues/947). 

Referring your comment and the test results I also had a look into the documentation:
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/radiogroup_role
As far as I understand the documentation, the arrow keys should be used to check/uncheck radio buttons inside a radio group

![image](https://github.com/user-attachments/assets/305938b1-c365-4b29-a66f-cb91aa90f1d3)

**Comment by GinaBiondo** [comment]
Hey @mmarinkov, sorry for the confusion. I misunderstood the comment in our open ticket. However for further alignment on this topic I would ask @carolabes to clarify what exactly the issue is :)

**Comment by GinaBiondo** [comment]
I think we can close the issue for now.
